clang/lib/Sema/SemaExprCXX.cpp

```diff
@@ -1082,6 +1082,10 @@ bool Sema::CheckCXXThrowOperand(SourceLocation ThrowLoc,
       // friendship or any other means).
       Context.addCopyConstructorForExceptionObject(Subobject, CD);
```

Comment:
It seems like we've already have the information so we don't need to add another bits for CXXRecordDecl, which is hot.

And generally we would write such information in the writer side and get such information and use it in the reader side.